karma.unit.test.conf.js

```diff
@@ -4,7 +4,13 @@ process.env.CHROME_BIN = puppeteer.executablePath();
 
 const configuration = {
   singleRun: true,
-  browsers: ['ChromeHeadless'],
+  browsers: ['ChromeHeadlessNoSandbox'],
+  customLaunchers: {
+    ChromeHeadlessNoSandbox: {
+      base: 'ChromeHeadless',
+      flags: ['--no-sandbox']
+    }
+  },
```

Comment:
I needed to use `no-sandbox` to run on the latest chrome.

Ref: https://github.com/karma-runner/karma-chrome-launcher/issues/158#issuecomment-339265457